assets/src/scripts/components/hydrograph/legend.spec.js

```diff
@@ -1,9 +1,10 @@
-import { select, selectAll } from 'd3-selection';
-import { drawSimpleLegend, legendMarkerRowsSelector, drawTimeSeriesLegend } from './legend';
-import { lineMarker, rectangleMarker, textOnlyMarker } from './markers';
+import {select, selectAll} from 'd3-selection';
+import {legendMarkerRowsSelector, drawTimeSeriesLegend} from './legend';
+import {drawSimpleLegend} from '../../d3-rendering/legend';
```

Comment:
This should be removed along with the tests for drawSimpleLegend. Those are in the d3-rendering/legend.spec.js file now.